docs/installer: update and include examples (#170)
kinvolk/lokomotive: docs/installer/lokoctl.md

### Download official releases

> Every release of lokoctl provides binary releases for several operating systems.

Reviewer: ```suggestion
Every release of Lokomotive provides `lokoctl` binary for several operating systems.
```

These binaries can be manually downloaded and installed.

1. Download your [desired version](https://github.com/kinvolk/lokomotive/releases), including the GPG
   signature.

2. Verify the signature. It should match one of the [Trusted
   keys](https://github.com/kinvolk/lokomotive/blob/master/docs/KEYS.md).

> ```

Reviewer 2: nit: ```console or ```bash

gpg --verify lokoctl_0.1.0_linux_amd64.tar.gz.sig
```

3. Unpack it

```
tar xvf lokoctl_0.1.0_linux_amd64.tar.gz
```

4. Find the lokoctl binary in the unpacked directory and move it to its desired location

```

> mv lokoctl_0.1.0_linux_amd64/lokoctl /usr/local/bin/lokoctl

Reviewer: Non-privileged users don't usually have access to write to `/usr/local/bin`. Should we maybe suggest `~/.local/bin`?

### Download official releases

> Every release of lokoctl provides binary releases for several OSes.

Reviewer 2: I would suggest

```suggestion
Every release of lokoctl provides binary releases for several operating systems.
```

These binary versions can be manually downloaded and installed.

1. Download your [desired version](https://github.com/kinvolk/lokomotive/releases), including the GPG
   signature.
2. Verify the signature. It should match one of the [Trusted
   keys](https://github.com/kinvolk/lokomotive/blob/master/docs/KEYS.md).

>     1. For example: `gpg --verify lokoctl_0.1.0_linux_amd64.tar.gz.sig`.

Reviewer 2: numbering here doesn't make sense.
```suggestion
    For example: `gpg --verify lokoctl_0.1.0_linux_amd64.tar.gz.sig`.
```

3. Unpack it

>     1. For example: `tar xvf lokoctl_0.1.0_linux_amd64.tar.gz`.

Reviewer 2: numbering here doesn't make sense.

```suggestion
    For example: `tar xvf lokoctl_0.1.0_linux_amd64.tar.gz`.
```

4. Find the lokoctl binary in the unpacked directory and move it to its desired location

>     1. For example. `mv lokoctl_0.1.0_linux_amd64/lokoctl /usr/local/bin/lokoctl`.

Reviewer 2: numbering here doesn't make sense.
```suggestion
    For example. `mv lokoctl_0.1.0_linux_amd64/lokoctl /usr/local/bin/lokoctl`.
```
